jenkins/run

```diff
@@ -33,13 +30,8 @@ if [[ -z "${IMAGE_REPO}" ]]; then
     IMAGE_REPO="quay.io/pbench"
 fi
 
-IHOME=/container/home
-
-if [[ -z "${CONTAINER_HOME}" ]]; then
-    HOME_VOLUME=${IHOME}
-else
-    HOME_VOLUME="${CONTAINER_HOME}:${IHOME}:${CONTAINER_HOME_OPTION:="z"}"
-fi
+USER_NAME=developer
+HOME_DIR=/home/${USER_NAME}
```

Comment:
I thought I'd commented on this before, but I can't find it.

Mostly, it's amusing rather than distressing that you undid the ability to map the transient container HOME into a host path; that was something Webb suggested as a "one plus" and I thought might possibly be useful. I don't bemoan the loss, although I wonder if we'd find a use for it.

I don't see any advantage of `/home/${USER_NAME}` over `/container/home` or something else equally arbitrary since the container is really single-user; but it doesn't hurt. It makes the test containers a bit less consistent, but I doubt that matters in any practical way.